Cargo.lock

```diff
@@ -2009,6 +2021,12 @@ version = "1.3.0"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "830d08ce1d1d941e6b30645f1a0eb5643013d835ce3779a5fc208261dbe10f55"
 
+[[package]]
+name = "leb128"
+version = "0.2.5"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "884e2677b40cc8c339eaefcb701c32ef1fd2493d71118dc0ca4b6a736c93bd67"
+
```

Comment:
As far as I can tell, `leb128` is maintained and published by the same org as `gimli`, under the same Apache2/MIT license.

(`gimli` previously used `leb128` as a dependency, but nowadays it uses its own slightly-modified copy in a submodule.)